python/ray/data/context.py

```diff
@@ -51,9 +51,9 @@
 # TODO (kfstorm): Remove this once stable.
 DEFAULT_ENABLE_PANDAS_BLOCK = True
 
-# Minimum amount of parallelism to auto-detect for a dataset. Note that the min
+# Minimum number of output blocks for a dataset. Note that the min
 # block size config takes precedence over this.
-DEFAULT_MIN_PARALLELISM = 200
+DEFAULT_MIN_NUM_BLOCKS = 200
```

Comment:
Not an issue, but these change will likely conflict with https://github.com/ray-project/ray/pull/43578. If this PR merges first, I'll update the other PR accordingly

Reply:
Cool, thanks for heads up. If https://github.com/ray-project/ray/pull/43578 merged first, I will update as well.